README.md

```diff
@@ -13,10 +13,11 @@
 - spreadsheet_id (string, required): your spreadsheet's id
 - worksheet_gid (integer, default: 0): worksheet's gid if you want to specific worksheet
 - mode (string, default: append): writing record method, available mode are `append` and `replace`
-- is_write_header (bool, default: false): if true, write header to first record
+- is_write_header_line (bool, default: false): if true, write header to first record
```

Comment:
http://www.embulk.org/docs/built-in.html#csv-formatter-plugin uses `header_line`. I think that having the same name with such standard plugins is better.